src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -106,62 +134,64 @@ export default class PhotoField extends React.Component {
     }
   }
 
-  componentDidUpdate(prevProps, prevState) {
-    const newView = this.state.src && (prevState.src !== this.state.src || (prevState.done && !this.state.done));
-    const cropper = this.refs.cropper;
-    if (cropper && newView) {
-      this.setCropBox();
-    }
-  }
-
   onEdit = () => {
     this.setState({ done: false, warningMessage: null });
   }
 
   onDone = () => {
-    this.setState({ done: true, warningMessage: null });
+    const cropResult = this.refs.cropper.getCroppedCanvas().toDataURL();
```

Comment:
To prevent the image from being resized and set to another type we can provide the original type here. This is done in the pending PR for triggering the file upload and is explained [here](https://github.com/fengyuanchen/cropperjs#known-issues).